### DISCUSSION TITLE
Getting a package to install

### DISCUSSION THREAD

**Discussion by giltonn** [community/community#148125] [New to GitHub]
I cannot get any package to install on my computer?

**Comment by pathaugen** [comment]
Which one do you want?

I'd make a folder on your computer, open command prompt or terminal and perform the command:

> git clone https://github.com/gdamore/tcell.git

Where the .git file is anything you wish to get on your computer.

<img width="748" alt="image" src="https://github.com/user-attachments/assets/2fbbfb45-c88d-41c3-937d-02196134d77e" />
